.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+
+node_js:
+  - '8'
+
+cache:
+  yarn: true
+  directories:
+    - node_modules
+
+notifications:
+  email: false
+
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.3.2
+  - export PATH=$HOME/.yarn/bin:$PATH
+
+script:
+  - yarn prettier:diff
```

Comment:
Might be helpful to match the react repo and expose a `ci-check` task that is run and holds all of these steps so that way people can run the ci check locally if they want.

Reply:
OK makes sense - Will take another look at this space.